### PR TITLE
Adding metadata keywords in MapDatasetEventSampler

### PR DESCRIPTION
**Pull request by fabiopintore**
The keywords of the event lists created by the `MapDatasetEventSampler` are not complete yet. This PR aims at adding all the necessary metadata that are still missing.

### CODE REVIEW

File: gammapy/datasets/simulate.py [hunk 1]
```diff
@@ -209,41 +209,61 @@ def event_list_meta(dataset, observation):
...
+        # Necessary for DataStore, but they should be ALT and AZ instead!
+        meta["ALTITUDE"] = 90
+        meta["ALT_PNT"] = 90
+        meta["AZ_PNT"] = 0
+
+        # TO DO: these keywords should be taken from the IRF of the dataset
+        meta["ORIGIN"] = "Gammapy"
+        meta["CALDB"] = ""
+        meta["IRF"] = ""
+        meta["TELESCOP"] = "CTA"
```
[the remark this answers is not in the document]
Comment (fabiopintore) [reply gammapy/datasets/simulate.py]: Done.

File: gammapy/datasets/simulate.py [hunk 2]
```diff
@@ -209,41 +209,61 @@ def event_list_meta(dataset, observation):
...
+        meta["DSUNI1"] = "s"
+        meta["DSVAL1"] = "TABLE"
+        meta["DSREF1"] = ":GTI"
+        meta["DSTYP2"] = "ENERGY"
+        meta["DSUNI2"] = "TeV"
+
+        meta["OBJECT"] = dataset.models.names[1]
+        meta["RA_OBJ"] = dataset.models[1].position.icrs.ra.deg
+        meta["DEC_OBJ"] = dataset.models[1].position.icrs.dec.deg
+        meta["TELAPSE"] = dataset.gti.time_sum.value
```
[the remark this answers is not in the document]
Comment (fabiopintore) [reply gammapy/datasets/simulate.py]: Done.

File: gammapy/datasets/simulate.py [hunk 3]
```diff
@@ -209,41 +209,61 @@ def event_list_meta(dataset, observation):
...
+
+        meta["OBSERVER"] = "Gammapy user"
+        meta["DSTYP1"] = "TIME"
+        meta["DSUNI1"] = "s"
+        meta["DSVAL1"] = "TABLE"
+        meta["DSREF1"] = ":GTI"
+        meta["DSTYP2"] = "ENERGY"
+        meta["DSUNI2"] = "TeV"
+
+        meta["OBJECT"] = dataset.models.names[1]
```
[the remark this answers is not in the document]
Comment (fabiopintore) [reply gammapy/datasets/simulate.py]: Sure, no problem to add the example.
However, do we think that all users will be interested in setting the OBJECT keyword? I would suspect that most of them would be already happy with the default value. What if we leave for now the first model as default, and we also provide an example in the tutorial showing how to set this keyword? On the other hand, this approach won't work if the dataset has only the background model...

[the remark this answers is not in the document]
Comment (adonath) [reply gammapy/datasets/simulate.py]: I think the case of background model only simulations is rare. If you clearly document the behaviour, I'm fine to keep the first source as default as well. Users can always overwrite it later...

**Pull request merged**